sdks/java/testing/test-utils/src/test/java/org/apache/beam/sdk/testutils/jvmverification/JvmVerification.java

```diff
@@ -70,6 +70,12 @@ public void verifyTestCodeIsCompiledWithJava21() throws IOException {
   }
 
   // jvm
+  @Test
+  public void verifyRunningJVMVersionIs8() {
```

Comment:
Why do we need this one?

Reply:
And should it replace the Is11 check?

Reply:
> Why do we need this one?

there is a `:sdks:java:testing:test-utils:verifyJavaVersion$testVer` target when gradle command has `testJavaVersion` property, added as dependency in a few tests here: https://github.com/apache/beam/blob/dbd719ba1448c13bc97237d1b701e2978c0e29d4/build.gradle.kts#L732

in order to verify the test indeed running on Java runtime of the version assigned by `-PtestJavaVersion=testVer`. Previously GHA test suites only has 11,17,21. Now this PR added a few Java8 tests with `-PtestJavaVersion=8` so new test is exercised.

> And should it replace the Is11 check?

I'm trying to generalize the buildSrc such that it does not assume the Java platform. Then one can run a test, saying, with default Java being Java8 or 11 and set `-PtestJavaVersion=17`. So I added a java8 test entry instead of replace 11 to 8.

Reply:
Got it, thanks - that makes sense.